### PR TITLE
Updated to use pug instead of jade

### PR DESCRIPTION
**Pull request by lakakucharita**
Came across the example and tried setting it up using Pug. This is in light of the total replacement of Jade by Pug.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,13 +16,13 @@ To build this example you will need to install the following two node packages:
...
 
-[Jade](https://github.com/visionmedia/jade)
+[Pug](https://github.com/pugjs/pug)
 
 ## Installation
 Then you can download this repository with
 
 ```sh
-git clone https://github.com/mattyod/matic-draft4-example
+git clone https://github.com/lakakucharita/matic-draft4-example
```
Comment (mattyod) [inline README.md]: Can you revert this line please?

File: README.md [hunk 2]
```diff
@@ -1,3 +1,8 @@
+Lakakucharita's notes:
+======================
+
+This is a fork of the original [example Matic project](https://github.com/mattyod/matic-draft4-example) that has been updated to use Pug instead of Jade.
```
Comment (mattyod) [inline README.md]: Can you revert this line and the title above please?

**Review by mattyod — CHANGES_REQUESTED** [review]
Thanks, just a couple of small things & I'lm happy to merge.

**Comment by lakakucharita** [comment]
Should be updated now. Kindly review and let me know if we're good to merge. Thanks!

**Comment by mattyod** [comment]
Thank you for the PR. RIP Jade 😢

**Pull request merged**